api/src/types/s2-api.ts

```diff
@@ -13,21 +13,52 @@ interface S2ApiSuccess {
   status: 200;
 }
 
-export interface S2ApiPaper {
-  abstract: string;
-  arxivId: string | null;
-  authors: S2ApiAuthor[];
-  doi: string;
-  title: string;
+export interface S2Api {
```

Comment:
nit pick: `S2Api` feels a bit too vague for this interface name, as it would also describe other calls that we make to the S2 API for data that are not papers. Can we give another name, say `S2ApiPaperResponse`?